starknet_py/contract.py

```diff
@@ -415,6 +483,81 @@ async def from_address(
 
         return Contract(address=address, abi=abi, client=client)
 
+    @staticmethod
+    async def declare(
+        account: AccountClient,
+        compiled_contract: str,
+        *,
+        max_fee: Optional[int] = None,
+        auto_estimate: bool = False,
+    ) -> DeclareResult:
+        """
+        Declares a contract
+
+        :param account: An AccountClient used to sign and send declare transaction.
+        :param compiled_contract: String containing compiled contract.
+        :param max_fee: Max amount of Wei to be paid when executing transaction.
+        :param auto_estimate: Use automatic fee estimation (not recommended, as it may lead to high costs).
+        :return: DeclareResult instance
+        """
+        declare_tx = await account.sign_declare_transaction(
+            compiled_contract=compiled_contract,
+            max_fee=max_fee,
+            auto_estimate=auto_estimate,
+        )
+        res = await account.declare(transaction=declare_tx)
+
+        return DeclareResult(
+            hash=res.transaction_hash,
+            _client=account.client,
+            class_hash=res.class_hash,
+            account=account,
+            compiled_contract=compiled_contract,
+        )
+
+    @staticmethod
+    async def deploy_contract(
```

Comment:
`declare_class` and `deploy_from_class`? I think this suggests logical flow of operations.

Reply:
@cptartur @Arcticae what do you think?

Reply:
I think `declare_class` is unecessary, since we have `declare` for a while now and it's kind of obvious what it does. 

`deploy_from_class` seems kind of redundant - all kinds of deploys will be based on class_hash soon so it doesn't make sense to specify that.

Reply:
But do we plan to remove .deploy before this version is deployed?

Reply:
I think so, yes. Deploy will most likely removed soon anyways and it's been deprecated for like good 1.5 month now.